### PR TITLE
Add list api

### PR DESCRIPTION
**Pull request by soroshsabz**
ITNOA

this PR provides list of all rules from system

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,8 @@
...
-
-This project powered on [IPTables.Net](https://github.com/splitice/IPTables.Net) for interacting with Linux IPTables
+# IpTables.Api
+API for local iptables management
+
+IpTables.Api is the RESTful server which applies iptables rules to block or allow IP addresses on request from a remote client.
+IpTables.Api maintains the list of blocked IP addresses which may be updated on the fly from many sources.
+IpTables.Api also solves the problem of concurrent modifications to iptables since the requests are serialized.
+
+This project powered on [IPTables.Net](https://github.com/splitice/IPTables.Net) for interacting with Linux IPTables
```
Comment (siadatism) [inline README.md:8]: "is powered"
or
"uses"

File: Source/BSN.IpTables.Data/IpTablesDotNetSystem.cs [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using SystemInteract.Local;
+
+namespace BSN.IpTables.Data
+{
+    public class IpTablesDotNetSystem : IIpTablesSystem
```
Comment (siadatism) [inline Source/BSN.IpTables.Data/IpTablesDotNetSystem.cs:13]: نام کلاس نامانوس به نظر می‌رسد

**Review by siadatism — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (soroshsabz) [reply README.md:8]: Done

File: Source/BSN.IpTables.Data/IpTablesDotNetSystem.cs (hunk 2 above)
Comment (soroshsabz) [reply Source/BSN.IpTables.Data/IpTablesDotNetSystem.cs:13]: Please give me some good name :(

**Pull request merged**